app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -728,31 +728,35 @@ def parse_base_item(item)
     def parse_image_name(image, image_ref)
       # parsing using same logic as in docker
       # https://github.com/docker/docker/blob/348f6529b71502b561aa493e250fd5be248da0d5/reference/reference.go#L174
-      parts = %r{
+      image_definition_re = %r{
         \A
+          (?<protocol>#{ContainerImage::DOCKER_PREFIXES.join('|')})?
           (?:(?:
             (?<host>([^\.:/]+\.)+[^\.:/]+)|
             (?:(?<host2>[^:/]+)(?::(?<port>\d+)))|
             (?<localhost>localhost)
           )/)?
           (?<name>(?:[^:/@]+/)*[^/:@]+)
-          (?:(?::(?<tag>.+))|(?:\@(?<digest>.+)))?
+          (?::(?<tag>[^:/@]+))?
+          (?:\@(?<digest>.+))?
         \z
-      }x.match(image)
+      }x
+      image_parts = image_definition_re.match(image)
+      image_ref_parts = image_definition_re.match(image_ref)
 
-      hostname = parts[:host] || parts[:host2] || parts[:localhost]
+      hostname = image_parts[:host] || image_parts[:host2] || image_parts[:localhost]
       [
         {
-          :name          => parts[:name],
-          :tag           => parts[:tag],
-          :digest        => parts[:digest],
+          :name          => image_parts[:name],
+          :tag           => image_parts[:tag],
+          :digest        => image_parts[:digest] || image_ref_parts[:digest],
```

Comment:
```
docker-pullable://registry.access.redhat.com/openshift3/metrics-deployer@sha256:45f2f81608edcdf53f8f94b4893fc33c824a5ca79d467191e5db1a267f46bb3a
```
It seems that the new docker added the registry and name to the imageID field too after the `docker-pullable://` prefix.